revenuecatui/src/iosMain/kotlin/com/revenuecat/purchases/kmp/ui/revenuecatui/PaywallOptionsKtx.kt

```diff
@@ -80,14 +75,4 @@ internal class IosPaywallDelegate(
     ) {
         listener?.onRestoreError(didFailRestoringWithError.toPurchasesErrorOrThrow())
     }
-
-
-    override fun paywallViewController(
-        controller: RCPaywallViewController,
-        didChangeSizeTo: CValue<CGSize>
-    ) {
-        var height: Int? = null
-        memScoped { height = didChangeSizeTo.ptr.pointed.height.toInt() }
-        onHeightChange(height!!)
-    }
```

Comment:
This is no longer needed because we set the height based on the `intrinsicContentSize` now. `didChangeSizeTo` fires multiple times with wildly different values, only the last one being correct. `intrinsicContentSize` only fires once, immediately with the right value.

Reply:
Hmm I'm a bit concerned... Would the kotlin side react to changes in the native view? I'm thinking specifically when changing from the "loading" UI to the "loaded" UI, the height is likely to change and not sure if the current alternative will actually update it then?

Reply:
You're totally right, thanks for catching this! 

This made me dig even deeper, and I now have an even better solution haha. We're now reporting the right height (from UIKit) immediately in the measurement phase. This means we can use `wrapContentHeight()`, which wasn't working before, and also allows us to keep using `animateContentSize()`. The animation is now only used to animate from loading to loaded, when the size changes, not when it first appears (because we already know the right size at that point). https://github.com/RevenueCat/purchases-kmp/pull/247/commits/da77881643968b97277bc4b116429425f34f57eb

Here's what it looks like when the offering needs to load (first one), and when it is already loaded and passed in (second one):

https://github.com/user-attachments/assets/deb877a1-aa93-4d66-9032-2f618c83abba